src/server/project.ts

```diff
@@ -657,7 +694,7 @@ namespace ts.server {
             if (this.lastReportedFileNames && lastKnownVersion === this.lastReportedVersion) {
                 // if current structure version is the same - return info without any changes
                 if (this.projectStructureVersion === this.lastReportedVersion && !updatedFileNames) {
-                    return { info, projectErrors: this.projectErrors };
+                    return { info, projectErrors: this.getGlobalProjectErrors() };

```

Comment:
Can you explain the change here? It's gone from erporting every error to returning only errors with no associated file. (What errors would those be?)